### PR TITLE
[Bugfix][CI/Build] Fix testing for generated commit hash

### PR DESCRIPTION
**Pull request by mgoin**

### CODE REVIEW

**Comment by youkaichao** [comment]
@mgoin  can you take a look at https://github.com/vllm-project/vllm/pull/6420 ?

File: .buildkite/test-pipeline.yaml [hunk 1]
```diff
@@ -194,7 +194,9 @@ steps:
   command: pytest -v -s test_logits_processor.py
 
 - label: Utils Test
-  command: pytest -v -s test_utils.py
+  commands:
+    - pytest -v -s test_utils.py
+    - pytest -v -s test_embedded_commit.py.py
```
Comment (DarkLight1337) [inline .buildkite/test-pipeline.yaml:199]:
```suggestion
    - pytest -v -s test_embedded_commit.py
```

**Comment by youkaichao** [comment]
@DarkLight1337  I think #6420 should be merged, and this should be closed.

**Comment by DarkLight1337** [comment]
Closing as superseded by #6420.

**Pull request closed without merging**